airgun/views/oscappolicy.py

```diff
@@ -32,6 +33,32 @@ def is_displayed(self):
             self.title, exception=False) is not None
 
 
+class ScapPolicyRadioGroup(RadioGroup):
+    """Handle an HTML non normalized Radio group according to the current
+    architecture.
+
+    Note: This is a temporary solution, a fix will be issued upstream,
+        when the fix will be available downstream we should replace the
+        implementation with RadioGroup.
+
+    Example html representation::
+
+        <div>
+        <input type="radio" value="ansible" name="policy[deploy_by]" id="policy_deploy_by_ansible">
+    """
+    @property
+    def button_names(self):
+        return ['ansible', 'puppet', 'manual']
+
+    def _get_parent_label(self, name):
+        """Get radio group label for specific button"""
+        return self.browser.wait_for_element(
+                f".//input[@id='policy_deploy_by_{name}']/.."
+            )
+        #  wait_for_element will raise NoSuchElementException when element is not found,
+        # so we don't need to raise it ourselves
```

Comment:
If you got a chance before someone merges that:
* this comment was an explanation for you, we don't really need it here :)
* indentation here looks weird, I would expect closing `)` to be on the same level as `return`. But if black doesn't complain, I guess that's fine.

Reply:
Thanks. Didn't noticed that. :P 
Fixed the code now.